src/server/ua_services_view.c

```diff
@@ -1250,7 +1250,7 @@ browseSimplifiedBrowsePath(UA_Server *server, const UA_NodeId origin,
     bp.relativePath.elementsSize = browsePathSize;
 
     /* Browse */
-    UA_UInt32 nodeClassMask = UA_NODECLASS_OBJECT | UA_NODECLASS_VARIABLE;
```

Comment:
The is is somewhat arbitrary... We could simplfy allow any node class here.
So just `nodeClassMask = ~0;`.

Reply:
I'am fine with taken all Nodeclasses into account. I will update the PR now.